### PR TITLE
chore: add translations for fr_FR

### PR DESCRIPTION
**Pull request by amine4567**
This add french (France) localization

### CODE REVIEW

**Comment by wescopeland** [comment]
Hi @amine4567! Happy to take what you already have. Not sure if you'd like to go over the latest en_US file before or after merging this one. With the introduction of the `<Trans />` component, several keys have changed.

**Comment by amine4567** [comment]
@wescopeland I updated the translations with the new keys

File: lang/fr_FR.json [hunk 1]
```diff
@@ -0,0 +1,416 @@
...
+    "Release Date": "Date de sortie",
+    "Remove from Want to Play Games": "Retirer des jeux à jouer",
+    "Removed {{gameTitle}} from playlist!": "{{gameTitle}} retiré de la liste !",
+    "Removed {{gameTitle}}!": "{{gameTitle}} retiré !",
+    "Removing...": "Retrait...",
+    "Reorder Site Awards": "Réorganiser les récompenses du site",
+    "Report Issue": "Signaler un problème",
+    "Report Issue - {{achievementTitle}}": "Signaler un problème - {{achievementTitle}}",
+    "Report to DevCompliance": "Signaler à DevCompliance",
+    "Report to QATeam": "Signaler à l'équipe QA",
```
Comment (HardSonicStorm) [inline lang/fr_FR.json:241]: La traduction devrait rester "QATeam" puisqu'il s'agit d'un compte
```suggestion
    "Report to QATeam": "Signaler à QATeam",
```

File: lang/fr_FR.json [hunk 2]
```diff
@@ -0,0 +1,416 @@
...
+    "Learn more about this event": "En savoir plus sur cet événement",
+    "Legal & Terms": "Mentions légales & Conditions",
+    "Less": "Moins",
+    "Load more": "Charger plus",
+    "Loading...": "Chargement...",
+    "Locale": "Langue",
+    "Manage Hashes": "Gérer les empreintes",
+    "Mastered": "Maîtrisé",
+    "Message": "Message",
+    "Message QATeam": "Message à l'équipe QA",
```
Comment (HardSonicStorm) [inline lang/fr_FR.json:172]:
```suggestion
    "Message QATeam": "Écrire à QATeam",
```

File: lang/fr_FR.json [hunk 3]
```diff
@@ -0,0 +1,416 @@
...
+    "Dev": "Dev",
+    "DevQuest": "DevQuest",
+    "Developer Compliance": "Conformité Développeur",
+    "Disclaimer about ROMs": "Avertissement concernant les ROMs",
+    "Disclaimers": "Avertissements",
+    "Documentation": "Documentation",
+    "Download Emulator": "Télécharger l'émulateur",
+    "Download Patch File": "Télécharger le fichier de correctif",
+    "Earliest": "Le plus tôt",
+    "Either": "Soit",
```
Comment (HardSonicStorm) [inline lang/fr_FR.json:120]: Dépend du contexte, si c'est un paramètre il vaudrait mieux utiliser "Peu importe" ou "N'importe lequel"

File: lang/fr_FR.json [hunk 4]
```diff
@@ -0,0 +1,416 @@
...
+    "Comments - {{user}}": "Commentaires - {{user}}",
+    "Comments on a forum topic I'm involved in": "Commentaires sur un sujet de forum auquel je participe",
+    "Comments on an achievement I created": "Commentaires sur un succès que j'ai créé",
+    "Comments on my activity": "Commentaires sur mon activité",
+    "Comments on my user wall": "Commentaires sur mon mur utilisateur",
+    "Comments: {{leaderboardTitle}}": "Commentaires : {{leaderboardTitle}}",
+    "Completed": "Terminé",
+    "Confirm New Email Address": "Confirmer la Nouvelle Adresse Email",
+    "Confirm Password": "Confirmer le Mot de Passe",
+    "Connect API Key": "Clé API Connect",
```
Comment (HardSonicStorm) [inline lang/fr_FR.json:88]:
```suggestion
    "Connect API Key": "Clé API Connecté",
```

File: lang/fr_FR.json [hunk 5]
```diff
@@ -0,0 +1,416 @@
...
+    "All Users": "Tous les Utilisateurs",
+    "All won achievements for this game": "Tous les succès remportés pour ce jeu",
+    "All-time High: {{val, number}} ({{date}})": "Record Absolu : {{val, number}} ({{date}})",
+    "Allow Comments on My User Wall": "Autoriser les Commentaires sur Mon Mur Utilisateur",
+    "Are you sure you want to cancel your request for account deletion?": "Êtes-vous sûr de vouloir annuler votre demande de suppression de compte ?",
+    "Are you sure you want to change your email address?": "Êtes-vous sûr de vouloir changer votre adresse e-mail ?",
+    "Are you sure you want to permanently delete all comments on your wall?": "Êtes-vous sûr de vouloir supprimer définitivement tous les commentaires sur votre mur ?",
+    "Are you sure you want to permanently delete this comment?": "Êtes-vous sûr de vouloir supprimer définitivement ce commentaire ?",
+    "Are you sure you want to request account deletion?": "Êtes-vous sûr de vouloir demander la suppression de votre compte ?",
+    "Are you sure you want to reset this progress? This cannot be reversed.": "Êtes-vous sûr de vouloir réinitialiser ce progrès ? Cela ne peut pas être annulé.",
```
Comment (HardSonicStorm) [inline lang/fr_FR.json:42]:
```suggestion
    "Are you sure you want to reset this progress? This cannot be reversed.": "Êtes-vous sûr de vouloir réinitialiser cette progression ? Cela ne peut pas être annulé.",
```

File: lang/fr_FR.json [hunk 6]
```diff
@@ -0,0 +1,416 @@
...
+    "Descending (Z - A)": "Descendant (Z - A)",
+    "Details on how the hash is generated for each system can be found <1>here</1>.": "Les détails sur la génération de l'empreinte pour chaque système se trouvent <1>ici</1>.",
+    "Dev": "Dev",
+    "DevQuest": "DevQuest",
+    "Developer Compliance": "Conformité Développeur",
+    "Disclaimer about ROMs": "Avertissement concernant les ROMs",
+    "Disclaimers": "Avertissements",
+    "Documentation": "Documentation",
+    "Download Emulator": "Télécharger l'émulateur",
+    "Download Patch File": "Télécharger le fichier de correctif",
```
Comment (HardSonicStorm) [inline lang/fr_FR.json:118]: Les patchs sont principalement pour les hacks et subsets, rarement pour corriger des problèmes
```suggestion
    "Download Patch File": "Télécharger le fichier de patch",
```

File: lang/fr_FR.json [hunk 7]
```diff
@@ -0,0 +1,416 @@
...
+    "Open game details": "Ouvrir les détails du jeu",
+    "Our web pages and their contents are subject to copyright law. Unless expressly permitted by law, every form of utilizing, reproducing or processing works subject to copyright protection on our web pages requires the prior consent of the respective owner of the rights. Individual reproductions of a work are allowed only for private use. The company names, product names, service names and logotypes on this web site are for identification purposes only. All trademarks and registered trademarks are the property of their respective owners.": "Nos pages web et leur contenu sont soumis au droit d'auteur. Sauf autorisation expresse de la loi, toute forme d'utilisation, reproduction ou traitement des œuvres protégées par le droit d'auteur sur nos pages web nécessite le consentement préalable du titulaire des droits. Les reproductions individuelles d'une œuvre sont autorisées uniquement pour un usage privé. Les noms de société, noms de produits, noms de services et logos sur ce site sont uniquement destinés à l'identification. Toutes les marques et marques déposées sont la propriété de leurs propriétaires respectifs.",
```
Comment (HardSonicStorm) [inline lang/fr_FR.json:205]:
```suggestion
    "Our web pages and their contents are subject to copyright law. Unless expressly permitted by law, every form of utilizing, reproducing or processing works subject to copyright protection on our web pages requires the prior consent of the respective owner of the rights. Individual reproductions of a work are allowed only for private use. The company names, product names, service names and logotypes on this web site are for identification purposes only. All trademarks and registered trademarks are the property of their respective owners.": "Nos pages web et leur contenus sont soumis au droits d'auteur. Sauf expressément autorisé par la loi, toute forme d'utilisation, reproduction ou traitement des œuvres protégées par le droit d'auteur sur nos pages web nécessite le consentement préalable du titulaire des droits. Les reproductions individuelles d'une œuvre sont autorisées uniquement pour un usage privé. Les noms de société, noms de produits, noms de services et logos sur ce site sont uniquement destinés à l'identification. Toutes les marques et marques déposées sont la propriété de leurs propriétaires respectifs.",
```

File: lang/fr_FR.json [hunk 8]
```diff
@@ -0,0 +1,416 @@
...
+    "Remove from Want to Play Games": "Retirer des jeux à jouer",
+    "Removed {{gameTitle}} from playlist!": "{{gameTitle}} retiré de la liste !",
+    "Removed {{gameTitle}}!": "{{gameTitle}} retiré !",
+    "Removing...": "Retrait...",
+    "Reorder Site Awards": "Réorganiser les récompenses du site",
+    "Report Issue": "Signaler un problème",
+    "Report Issue - {{achievementTitle}}": "Signaler un problème - {{achievementTitle}}",
+    "Report to DevCompliance": "Signaler à DevCompliance",
+    "Report to QATeam": "Signaler à l'équipe QA",
+    "Report to WritingTeam": "Signaler à l'équipe de rédaction",
```
Comment (HardSonicStorm) [inline lang/fr_FR.json:242]:
```suggestion
    "Report to WritingTeam": "Signaler à WritingTeam",
```

File: lang/fr_FR.json
Comment (HardSonicStorm) [inline lang/fr_FR.json:1]: Pour la traduction des "email", il y a des "e-mail" et "email", il faudrait choisir l'un des deux pour rester consistant.

File: lang/fr_FR.json (hunk 3 above)
Comment (amine4567) [reply lang/fr_FR.json:120]: Can't find where this is used. @wescopeland, depending on the context, `Either` can be translated in different ways

Comment (wescopeland) [reply lang/fr_FR.json:120]: This may actually be a dead string. I think we changed this to "Both". I will make a note that this should be removed.

**Review by wescopeland — APPROVED** [review]
Thanks!

**Pull request merged**